### ISSUE TITLE
examples/train_textcat.py fails to load model after training: AttributeError: 'FeedForward' object has no attribute 'W'

### ISSUE THREAD

**Issue by Solid7** [explosion/spacy-transformers#105]
I might have a bad installation but not sure where to start debugging this. When I run the example script 'examples/train_textcat.py' i get the following error.

**Steps to reproduce:**
```
python train_textcat.py -o output en_trf_distilbertbaseuncased_lg -bs 1 -t 20
```

**Output:**

```
Saved model to output
Loading from output
Traceback (most recent call last):
  File "pipes.pyx", line 218, in spacy.pipeline.pipes.Pipe.from_disk.load_model
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\thinc\neural\_classes\model.py", line 375, in from_bytes
    dest = getattr(layer, name)
AttributeError: 'FeedForward' object has no attribute 'W'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "train_textcat.py", line 283, in <module>
    plac.call(main)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\plac_core.py", line 328, in call
    cmd, result = parser.consume(arglist)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\plac_core.py", line 207, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)
  File "train_textcat.py", line 171, in main
    nlp2 = spacy.load(output_dir)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\spacy\__init__.py", line 30, in load
    return util.load_model(name, **overrides)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\spacy\util.py", line 219, in load_model
    return load_model_from_path(name, **overrides)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\spacy\util.py", line 262, in load_model_from_path
    return nlp.from_disk(model_path)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\spacy\language.py", line 940, in from_disk
    util.from_disk(path, deserializers, exclude)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\spacy\util.py", line 733, in from_disk
    reader(path / key)
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\spacy\language.py", line 935, in <lambda>
    p, exclude=["vocab"]
  File "pipes.pyx", line 227, in spacy.pipeline.pipes.Pipe.from_disk
  File "C:\Users\foo\Anaconda3\envs\py37\lib\site-packages\spacy\util.py", line 733, in from_disk
    reader(path / key)
  File "pipes.pyx", line 220, in spacy.pipeline.pipes.Pipe.from_disk.load_model
ValueError: [E149] Error deserializing model. Check that the config used to create the component matches the model being loaded.
```

Looks like loading the model from disk fails once it is trained.

Relevant packages
```
spacy 2.2.2
spacy-transformers 0.5.1
thinc 7.3.1
thinc-gpu-ops 0.0.4+cuda100
transformers 2.0.0
```

```
:> python -m spacy validate

TYPE      NAME                             MODEL                            VERSION
package   en-trf-distilbertbaseuncased-lg   en_trf_distilbertbaseuncased_lg   2.2.0     ✔
package   en-trf-bertbaseuncased-lg        en_trf_bertbaseuncased_lg        2.2.0     ✔
package   en-pytt-robertabase-lg           en_pytt_robertabase_lg           2.1.0     --> n/a
package   en-pytt-bertbaseuncased-lg       en_pytt_bertbaseuncased_lg       2.1.1     --> n/a
package   en-core-web-sm                   en_core_web_sm                   2.2.0     ✔
package   en-core-web-md                   en_core_web_md                   2.2.0     ✔
package   en-core-web-lg                   en_core_web_lg                   2.2.0     ✔

The following models are not available for spaCy v2.2.2: en_pytt_robertabase_lg,
en_pytt_bertbaseuncased_lg
```

**Comment by rollator3000** [comment]
Having the same Issue! 
Can't load a finetuned model --> same error as above: 
`"AttributeError: 'FeedForward' object has no attribute 'W'"`

Training/ FineTuning works, but reading in (& making use of) the fine tuned models doesn't work right now, as reading in fails!

Relevant packages:
```
spacy 2.2.2
spacy-transformers 0.5.1
thinc 7.3.0
thinc-gpu-ops 0.0.4+cuda100
transformers 2.0.0

```

Thanks in advance!

**Comment by pvcastro** [comment]
I'm also having the same problem!

```
spacy 2.2.2
spacy-transformers 0.5.1
thinc 7.3.1
thinc-gpu-ops 0.0.4+cuda101
transformers 2.0.0 
```

**Comment by wavymazy** [comment]
Hey guys I may have stumbled on a solution, try using spacy 2.2.1.

**Comment by Solid7** [comment]
big thanks @wavymazy! 🎊

I can confirm that @wavymazy suggestion to downgrade to spacy 2.2.1 works. Model is loaded without errors.

This is my working setup:
```
spacy 2.2.1 <--
spacy-transformers 0.5.1
thinc 7.1.1 <--
thinc-gpu-ops 0.0.4+cuda100 
transformers 2.0.0
```
Looks like my combination of thinc, spacy and spacy-transformers was not compatible.

**Comment by rollator3000** [comment]
Can also confirm, that @wavymazy solution of downgrading is working :tada: 

Thank you @wavymazy :heart_eyes:

**Comment by svlandeg** [comment]
@Solid7 @rollator3000 @pvcastro @wavymazy  : PR https://github.com/explosion/spacy-transformers/pull/120 should address this issue. Basically the problem is in the `meta.json` where it reads `"trf_textcat":"textcat"` instead of `"trf_textcat":"trf_textcat"`. While we're fixing the underlying issue, a quick hack is probably to go in and change the `meta.json` yourself in case you don't want to rerun the training script.

**Comment by hi-im-ryanli** [comment]
@svlandeg's hack worked for me, but I am wondering if the issue has been fixed? I am using spacy 2.2.4 and still ran into the issue.
